block/block.go

```diff
@@ -112,6 +113,30 @@ func (m *Manager) applyBlock(ctx context.Context, block *types.Block, commit *ty
 	return nil
 }
 
+func (m *Manager) checkPrevCachedBlocks(ctx context.Context) error {
```

Comment:
as this function is not only checking it's also applying I think the name is confusing. Also when it says "Prev" blocks it can be confusing as it has been previously cached but those blocks can be in the future (i.e not yet applied)

Reply:
name changed to attemptApplyCachedBlocks